app/src/main/java/dev/jay/aushadhi/ui/screens/splash/SplashScreen.kt

```diff
@@ -0,0 +1,95 @@
+package dev.jay.aushadhi.ui.screens.splash
+
+import androidx.compose.animation.core.FastOutSlowInEasing
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import dev.jay.aushadhi.R
+import dev.jay.aushadhi.ui.components.AushadhiScreen
+import dev.jay.aushadhi.ui.components.SplashScreen
+import dev.jay.aushadhi.ui.theme.OrangeSecondary
+import kotlinx.coroutines.delay
+
+@Composable
+fun SplashScreen(navController: NavController) {
```

Comment:
issue: Android 12 onwards, a default Splash Screen is attached to all apps. It's best to mod that screen rather than creating your own since it leads to user seeing 2 splash screens